debian/qubes-core-agent.postinst

```diff
@@ -186,8 +186,8 @@ case "${1}" in
             rm /etc/systemd/system/multi-user.target.wants/qubes-mount-home.service
         fi
 
-        if ! dpkg-statoverride --list /var/lib/qubes/dom0-updates >/dev/null 2>&1; then
-            dpkg-statoverride --update --add user user 775 /var/lib/qubes/dom0-updates
+        if ! dpkg-statoverride --list /var/lib/qubes/dom0-updates >/dev/null 2>&1 && user=$(qubesdb-read /default-user); then
```

Comment:
This needs to have a default to `user`, qubesdb may not function during package install yet (when building fresh template).

Reply:
ping?